packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -51,14 +51,15 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
 
             // Normalize relative import to absolute import
             if (importee.startsWith('.') && importer) {
+                const ext = path.extname(importee) || path.extname(importer);
```

Comment:
@wes566 why are you checking the extension of both importer and importee?

Reply:
So first it checks if the importee specifies an extensions... so if you had `import foo from ./foo.js` it would use `.js`... whereas if the import statement said `import foo from ./foo` then it will match the importer's extension.

Reply:
what happens if your entry file has a '.js' extension, however your imported file, by mistake, has a .ts extension? 
// entry.js
import foo from './foo' // foo points to foo.ts

Reply:
Ok what we are saying is that if the file you are in is TS, by we will try to match it, but if your relative file is for whatever weird reason.
So yes we should throw